composer.json

```diff
@@ -1,5 +1,5 @@
 {
-    "name": "box/spout",
```

Comment:
This shouldn't be changed in your PR

Reply:
Yes, as being said I needed this in prod, so I'll commit another with the correct package name